docs/content.zh/docs/connectors/pipeline-connectors/oceanbase.md

```diff
@@ -57,6 +57,8 @@ sink:
 pipeline:
   name: MySQL to OceanBase Pipeline
   parallelism: 1
+  flink:
+    execution.checkpointing.interval: 2min
```

Comment:
I added flink parameters to the pipeline-connector demo, prompting users to configure flink parameters in pipeline.yaml